### ISSUE TITLE
Possible error in NodeCluster.offset() method in NodeCluster class.

### ISSUE THREAD

**Issue by SteveLamont** [nabil6391/graphview#27]
It looks to me as if there is an error in setting the offset of the `rect` property in the `offset()` method in the `NodeCluster` class.

The code currently reads
```  void offset(double xDiff, double yDiff) {
    nodes.forEach((node) {
      node.position = (node.position + Offset(xDiff, yDiff));
    });

    rect.translate(xDiff, yDiff);
  }
```
This, I suggest, is incorrect as the `translate()` method of the Dart `Rect` object returns a new `Rect` object, not modifying the original `Rect`.

Thus, I believe the code should read
```  void offset(double xDiff, double yDiff) {

    nodes.forEach((node) {
      node.position = (node.position + Offset(xDiff, yDiff));
    });

    rect = rect.translate(xDiff, yDiff);
    
  }
```

**Comment by nabil6391** [comment]
Thanks,  I will have a look.

**Comment by SteveLamont** [comment]
Any ETA on a fix for this?

Thanx.

**Comment by nabil6391** [comment]
@SteveLamont Its done. Fixed in 0.6.7 . Was it causing any significant problem for you

**Comment by SteveLamont** [comment]
Thanx for the fix.

Yes, when multiple node clusters were present, they would be be positioned badly.
